src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,7 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery one = new Lottery();
+        one.getRandomBall(3);
```

Comment:
```suggestion
        Lottery lottery = new Lottery();
        for (int i = 0; i < 3; i++) {
            lottery.getRandomBall();
        }
```
lottery should return just 1 ball